internal/exec/stages/files/files.go

```diff
@@ -30,6 +33,10 @@ const (
 
 var (
 	ErrFilesystemUndefined = errors.New("the referenced filesystem was not defined")
+
+	// To enable relabeling, override at link-time with:
+	// -X github.com/coreos/ignition/internal/exec/stages/files.selinuxRelabel=enabled
+	selinuxRelabel = "disabled"
```

Comment:
when not use a bool?

Reply:
`-X` only works on string variables.

Reply:
Why would this ever be set on an SELinux system to disabled?  The restorecon command will check if SELinux is enabled and become a  noop.  So why disable this at all on RHEL/Fedora/Centos systems?

Reply:
That's correct re. `-X`; AFAICT you can only pass string values.

> Why would this ever be set on an SELinux system to disabled?

On Container Linux, SELinux is enabled but permissive. To be conservative, we're disabling this relabeling mechanism there for the time being.

Reply:
Why?  I actually think Permissive is the worse setting, Permissive gives customers the warm and fuzzies and then they ignore the errors.  We will not get feed back on SELinux breakage unless it is enforcing mode.

Reply:
CL tries to support people making the system enforcing optionally today; if it's disabled that's much harder as you have to relabel.

(Let's also not make fundamental changes to CL a requirement of this PR)

Reply:
And of course, part of the issue with enforcing mode today is exactly because of the issues that this very PR is trying to solve. So at the very least, any discussions around changing to enforcing mode requires issues addressed by this PR to already be resolved.

Reply:
All options that can be overridden at link time should go in the `internal/distro` package.

Reply:
(which would be a good place for a wrapper that returns `bool`)

Reply:
@rhatdan We effectively can't switch it without causing major breakage either. Since CL has automatic updates that happen without intervention we have to be incredibly sure our update won't break users. Switching from permissive to enforcing would do that (especially since the CL policy is somwhat of a mess). It would have been better to have it enforcing from the start, but we can't really fix that now.

Reply:
So does this mean til the end of time, SELinux needs to be disabled by default? That is unacceptable for an enterprise.

Reply:
For CL, yes. We cannot ship an update that knowingly breaks users. For FCOS and RHCOS we need to have it enabled, enforcing, and working from day 1.